lib/iex/endpoints/historial_prices.rb

```diff
@@ -0,0 +1,28 @@
+module IEX
+  module Endpoints
+    module HistoricalPrices
+      def historical_prices(symbol, options = {})
+        if options[:range] == 'date'
+          raise ArgumentError if options[:date].nil?
+          raise ArgumentError if options[:chartByDay].nil?
+        end
+
+        options = options.dup
+        path = "stock/#{symbol}/chart"
+        path += "/#{options[:range]}" if options[:range]
+        # IEX needs dates passed without any formatting, trim out common symbols if passed in.
+        path += "/#{options[:date].tr('-/', '')}" if options[:range] == 'date'
```

Comment:
This feels wrong. IEX expects a date in a well defined format, not one without dashes or slashes, and it's not for the client to be mangling input. It's a slippery slope when IEX adds support for `-5d` (last 5 days) or something like that and we have to deprecate magical behavior. So either leave the input as is, and document what's required, and/or add support for passing in a DateTime and convert it into the right kind of string.

Reply:
So i did this because typing in a date as `20201101` is a lot more difficult as a user of the library than typing in `2020-11-01` or `2020/11/01` as most humans would type a date. I like the DateTime approach more through so I'll change that and the documentation.

Reply:
On second thought, any specific reason for DateTime vs Date? This endpoint has no concept of time in its current implementation.